plugins/woocommerce-blocks/assets/js/blocks/checkout/address-card/index.tsx

```diff
@@ -25,6 +26,19 @@ const AddressCard = ( {
 	target: string;
 	fieldConfig: FormFieldsConfig;
 } ): JSX.Element | null => {
+	const formattedCountry =
+		typeof ALLOWED_COUNTRIES[ address.country ] === 'string'
+			? decodeEntities( ALLOWED_COUNTRIES[ address.country ] )
+			: '';
+
+	const formattedState =
+		typeof ALLOWED_STATES[ address.country ] === 'object' &&
```

Comment:
this is true for `null`, we do have an `isObject` helper and `objectHasProp` helper in [this file](https://github.com/woocommerce/woocommerce/blob/336591f8d4c9b1b41d079069f67516387566680c/plugins/woocommerce-blocks/assets/js/types/type-guards/object.ts#L7)

Reply:
<img width="196" alt="image" src="https://github.com/woocommerce/woocommerce/assets/5656702/f7d89e36-e215-420a-b2c1-8592217822fe">

Reply:
Okay, I will update the other place in which this happens as well.